.pipelines/templates/e2e-windows-template.yaml

```diff
@@ -31,7 +31,8 @@ jobs:
         export WINDOWS_E2E_IMAGE=${{ parameters.windowsImage }}
         export WINDOWS_E2E_OSSKU=${{ parameters.windowsOSSKU }}
         export WINDOWS_E2E_VMSIZE=${{ parameters.windowsVMSize }}
-        export WINDOWS_E2E_STORAGE_ACCOUNT=${{ parameters.storageAccount }}
+        K=${KUBERNETES_VERSION//./}
+        export WINDOWS_E2E_STORAGE_ACCOUNT=${{ parameters.storageAccount }}$K
```

Comment:
Suggest to create containers in existing storage accounts instead of creating storage accounts.

Reply:
Storage account will be created only once if the RG does not exist for the k8s version e.g. it has been GCed. (if boolean indicator `$create_cluster` is true in line 88, `e2e-starter.sh`)

In subsequent testing on the same k8s version, SA won't be created assuming the RG is still there.

Or by saying existing storage accounts are you referring to something else more specifically?

Reply:
Used shared storage account in the VHD image release sub now.